<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingwindow</class>
 <widget class="QWidget" name="settingwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GameSetting</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>451</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout" stretch="5,1,1">
    <property name="spacing">
     <number>8</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>25</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="selectLayout">
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="Line" name="line_8">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="leftLayout" stretch="0,0,0,0,0,0,0,0">
        <item>
         <widget class="Line" name="line_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="redSetLog">
          <property name="text">
           <string>RedPlayer(first)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="redHumanPolicyButton">
          <property name="text">
           <string>Human</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="redRandomPolicyButton">
          <property name="text">
           <string>Random</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="redMctsPolicyButton">
          <property name="text">
           <string>Bee</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="redNetPolicyButton">
          <property name="text">
           <string>Net</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="timeLayout" stretch="5,1">
          <property name="spacing">
           <number>5</number>
          </property>
          <item>
           <widget class="QLineEdit" name="timeSetLog">
            <property name="text">
             <string>Time(mins):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="timeSetBox">
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>3</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="rightLayout">
        <item>
         <widget class="Line" name="line_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="blueSetLog">
          <property name="text">
           <string>BluePlayer(second)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="blueHumanPolicyButton">
          <property name="text">
           <string>Human</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="blueRandomPolicyButton">
          <property name="text">
           <string>Random</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="blueMctsPolicyButton">
          <property name="text">
           <string>Bee</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="blueNetPolicyButton">
          <property name="text">
           <string>Net</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="sizeLayout">
          <item>
           <widget class="QLineEdit" name="sizeSetLog">
            <property name="text">
             <string>BoardSize:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sizeSetBox">
            <property name="minimum">
             <number>3</number>
            </property>
            <property name="maximum">
             <number>25</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_7">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="okLayout" stretch="2,3,2">
      <property name="leftMargin">
       <number>15</number>
      </property>
      <property name="rightMargin">
       <number>15</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="okButton">
        <property name="text">
         <string>Ok</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
